metadata-models/src/main/pegasus/com/linkedin/metadata/key/JoinKey.pdl

```diff
@@ -0,0 +1,18 @@
+namespace com.linkedin.metadata.key
+
+/**
+ * Key for a Join
+ */
+@Aspect = {
+  "name": "joinKey"
+}
+record JoinKey {
+  /*
+  * Unique guid for Join
+  */
+  @Searchable = {
+    "fieldType": "TEXT",
+  }
```

Comment:
do we need this to be searchable? it looks like right now it's just a random UUID and we're making the `name` searchable anyways for joins

Reply:
Yeah agreed... removed.